epub33/core/index.html

```diff
@@ -7428,7 +7428,7 @@ <h5>The <code>metadata</code> Element</h5>
 							</dd>
 						</dl>
 
-						<p>This specification defines no metadata properties that MUST occur in the Media Overlay
+						<p>This specification defines not require any metadata properties in the Media Overlay
```

Comment:
This line does not read all right... should it be

> This specification does not require any metadata properties...

Reply:
Oops, yes, should have s/defines/does/